Sources/SmithyEventStreamsAPI/EventStreamError.swift

```diff
@@ -15,11 +15,3 @@ public enum EventStreamError: Error {
     /// This may be due to missing required headers
     case invalidMessage(String)
 }
-
```

Comment:
The extension below was unused and causing compiler warnings, so it was deleted.